sys/net/application_layer/gcoap/fileserver.c

```diff
@@ -32,14 +32,24 @@
 /** Maximum length of an expressible path, including the trailing 0 character. */
 #define COAPFILESERVER_PATH_MAX (64)
 
+struct requestoptions {
+    struct {
+        bool if_match        :1;
+        bool if_none_match   :1;
+        bool etag            :1;
+        bool block2          :1;
+        bool block1          :1;
+    } exists;
+    uint32_t etag;
+    uint32_t if_match;
+    char if_match_len;
+};
+
 /** Data extracted from a request on a file */
 struct requestdata {
     /** 0-terminated expanded file name in the VFS */
     char namebuf[COAPFILESERVER_PATH_MAX];
-    uint32_t blocknum2;
-    unsigned int szx2; /* would prefer uint8_t but that's what coap_get_blockopt gives */
```

Comment:
Were did this disappear to.

Reply:
Block2 is now only handled in the GET method handlers. Now that other methods are supported, I did not see a need to reserve memory for that in `struct requestdata`.